generators/client/templates/angular/src/main/webapp/app/admin/metrics/metrics.component.html.ejs

```diff
@@ -47,8 +47,6 @@
     </div>
 
     <div *ngIf="metrics && metricsKeyExists('garbageCollector')">
-        <h3 <%= jhiPrefix %>Translate="metrics.jvm.gc.title">Garbage collector statistics</h3>
-
         <<%= jhiPrefixDashed %>-metrics-garbagecollector
             [updating]="updatingMetrics"
             [garbageCollectorMetrics]="metrics.garbageCollector">
```

Comment:
Now it looks cleaner without outer `div` like other components are used below, can you do also this polishing:
```
    <<%= jhiPrefixDashed %>-metrics-garbagecollector
        *ngIf="metrics && metricsKeyExists('garbageCollector')"
        [updating]="updatingMetrics"
        [garbageCollectorMetrics]="metrics.garbageCollector">
    </<%= jhiPrefixDashed %>-metrics-garbagecollector>
```